mods/rv/rules/ai.yaml

```diff
@@ -920,9 +983,18 @@ Player:
 			txdx: 1
 			disk: 2
 			magnedisk: 1
+			zep: 1
+			kite: 1
+			havoc: 1
+			shad-squad-1: 1
+			shad-squad-2: 1
+			shad-squad-3: 1
```

Comment:
I wanna say these needs some UnitLimits, looks wierd with AI having so many Nighthawks, on other hand Nighthawks may survive after deploying their units and they maybe moving around empty and new ones wouldn't be build if that were the case.

Reply:
Limit is controlled by `~!LoadedExists`. Only helicopter unload all AI will build a new helicopter, which can control the number of helicopter while not affect normal transport tactical.

Reply:
Changed to all pre-loaded Nighthawks share a buildlimit lock with each other.